__tests__/index.test.js

```diff
@@ -114,10 +117,46 @@ describe("/api/articles", () => {
   test("GET:200 returns an array of articles sorted by created date descendingly", () => {
     return getRequest.then((response) => {
       const { articles } = response.body;
-      console.log(articles, "res");
       expect(articles).toBeSortedBy("created_at", {
         descending: true,
       });
     });
   });
 });
+
+// comments tests
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET:200 responds with all article comments sorted by created_at desc", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        const comments = response.body;
+        expect(comments).toBeSortedBy("created_at", {
+          descending: true,
+        });
+
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("GET:200 sends an empty array if there is no comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then((response) => {
+        const comments = response.body;
+        expect(comments.length).toBe(0);
+      });
+  });
+});
```

Comment:
These two happy path tests are really strong, well done! 🙌 
However we're missing some error handling here, namely
400: given an article that is invalid e.g. `/api/articles/katherine/comments`
404: given a valid article id but it doesn't exist e.g. `/api/articles/10000000/comments`
Could you please add in these tests and get them passing
Your happy path tests should stay passing too! 🔍

Reply:
I wonder if i have same tests for articles it’s not necessary?